index.html

```diff
@@ -40,36 +42,272 @@ <h2 id="jobTitle">I am a Software Developer</h2>
+      <path d="M88.8671 280.778L107.145 334.823H20.327L0.473961 280.778H88.8671Z" fill="white" stroke="#262626" stroke-width="0.787818" stroke-miterlimit="10"/>
+      <path d="M154.255 280.778C154.255 280.778 157.564 276.997 148.74 260.138C139.917 243.121 115.494 220.431 115.494 220.431C115.494 220.431 105.568 210.505 103.835 208.772L175.684 260.61L154.255 280.778Z" fill="#262626"/>
+      <circle r="11.625" transform="matrix(-1 0 0 1 53.2695 309)" fill="white" stroke="#262626" stroke-width="0.75"/>
+      </g>
+      <defs>
+      <clipPath id="clip0_34_1555">
+      <rect width="673.269" height="353.1" fill="white" transform="matrix(-1 0 0 1 673.27 0)"/>
+      </clipPath>
+      <clipPath id="clip1_34_1555">
+      <rect width="376.199" height="317.933" fill="white" transform="matrix(1 0 0 -1 220.07 286.933)"/>
+      </clipPath>
+      </defs>
+      </svg>
+      <h2 id="about-title">About me</h2>
+      <P id="about-text">Hello I’m a software developer! I can help you build a product , 
+          feature or website Look through some of my work and experience! 
+          If you like what you see and have a project you need coded, don’t  
+          hestiate to contact me.</P>
+          <a id="get-my-resume" href="#">Get My Resume</a>
+
+          <hr id="about-hr">
+          <div class="list_lang">
+              <ul><li class="li-first-child"><span><img class="lang-img-icon"src="./images/diamond.png" alt=""></span>
+                  Language</li>
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li></ul>
+
+              <ul><li class="li-first-child"><span><img class="lang-img-icon"src="./images/square.png" alt=""></span>
+                  Framworks</li>
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li></ul>
+
+              <ul><li class="li-first-child"><span><img class="lang-img-icon" src="./images/circle.png" alt=""></span>
+                  Skills</li>
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li></ul>
+          </div>
+        </div>
+        <div class="form_section">
+        <!-- <p id="get-in-touch-text">
+            I'm always interested in hearing 
+            about new projects, so if you'd
+             like to chat please get in touch.
+        </p> -->
+        <!-- <form action="">
+            <label for="name">
+                Full name <br>
+                <input type="text">
+            </label>
+            <label for="email">
+                Email address <br>
+                <input type="email">
+            </label>
+            <label for="commet-box">
+                <input type="textarea" placeholder="Enter text here">
+            </label>
+        </form> -->
```

Comment:
- [ ] Kindly make sure you remove commented code before tracking with version control(git). So please remove this commented code.